src/surfing_penguin/routes.py

```diff
@@ -4,6 +4,7 @@
 from flask import render_template, flash, redirect, url_for, request
 from flask_restplus import Resource
 from surfing_penguin.forms import LoginForm
+from surfing_penguin.models import Qstnr, question
 
```

Comment:
These lines did not pass the flake8 check.

Reply:
Please make sure the code could work and pass the style checker before pushing. This is the basic request. surfing_penguin.forms  doesn't even exist.